openstack/cronus/templates/nebula/secrets.yaml

```diff
@@ -2,16 +2,31 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: nebula
+  name: nebula-secret
   namespace: {{ .Release.Namespace }}
-  annotations:
-    cloud.sap/skip-template: "true"
   labels:
     app: nebula
     chart: nebula
     release: nebula
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  config.yaml: {{ include "cronus/templates/nebula/_config.yaml.tpl" . | b64enc }}
+  SENTRY_DSN: {{ .Values.nebula.sentryDsn | b64enc }}
```

Comment:
If I compare this with the config tpl above, most of these were wrapped in if statements:
`{{- if .Values.pki.enabled }}`
`{{- if .Values.notifier.enabled }}`
`{{- if .Values.nebula.sentryDsn }}`
`{{- if .Values.hermes }}`

But I think without these values cronus would never work. So having this in an if likely never made sense, as they are required properties. But @sapmli would know better

Reply:
Looking at the nebula PR, and it indeed won't work without these there, so lets leave it how you have it. I will resolve my comment